src/libsystemd-network/sd-dhcp-client-id.c

```diff
@@ -93,3 +93,67 @@ int sd_dhcp_client_id_set_iaid_duid(
         client_id->size = offsetof(typeof(client_id->id), ns.duid) + duid->size;
         return 0;
 }
+
+int sd_dhcp_client_id_to_string(const sd_dhcp_client_id *client_id, char **ret) {
+        _cleanup_free_ char *t = NULL;
+        size_t len;
+        int r;
+
+        assert_return(sd_dhcp_client_id_is_set(client_id), -EINVAL);
+        assert_return(ret, -EINVAL);
+
+        len = client_id->size - offsetof(typeof(client_id->id), data);
+
+        switch (client_id->id.type) {
+        case 0:
```

Comment:
no named enum values for the cases of this switch?

Reply:
I will add an enum for the type in a later PR.